index.html

```diff
@@ -3,10 +3,22 @@
   <head>
      <meta charset="UTF-8">
      <title>Angular ToDo App</title>
+     <link rel="stylesheet" href="style.css">
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
      <script src="app.js"></script>
   </head>
-  <body>
+  <body data-ng-app="todo">
     <h1>To Do!</h1>
+    <div data-ng-controller="todo_controller as vm">
+      <form name="addToDo">
+        <input type="text" data-ng-model="vm.formToDoText">
+        <button data-ng-click="vm.create()">Add</button>
+      </form>
+
+      <div ng-repeat="task in vm.tasks">
+        <span class="close" data-ng-click="vm.delete($index)">X</span>
+        <span class="done-{{task.done}}" value="{{task.task}}" contentEditable="true">{{ task.task }}</span>
```

Comment:
Not sure that you really need to edit or modify the todo text in this way using contentEditable, but it's ultimately up to you! Definitely nothing wrong with using it, just a thought.